Tests/Editor/GameLiftPlugin/AnywherePageTests.cs

```diff
@@ -0,0 +1,176 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Amazon.GameLift.Model;
+using AmazonGameLift.Editor;
+using AmazonGameLiftPlugin.Core.ApiGatewayManagement;
+using AmazonGameLiftPlugin.Core.SettingsManagement.Models;
+using Moq;
+using NUnit.Framework;
+
+namespace Editor.Window.UnitTests
+{
+    [TestFixture]
+    public class AnywherePageTests
+    {
+
+        private Mock<GameLiftPlugin> _gameLiftConfigMock;
+        private Mock<IAmazonGameLiftClientWrapper> _gameLiftWrapperMock;
+        private Mock<IAwsCredentialsFactory> _awsCredentialsFactory;
+        private Mock<CoreApi> _coreApi;
+        private Mock<IAmazonGameLiftClientFactory> _amazonGameLiftClientFactory;
+        
+        [SetUp]
+        public void Setup()
+        {
+            _gameLiftConfigMock = new Mock<GameLiftPlugin>();
+            _gameLiftWrapperMock = new Mock<IAmazonGameLiftClientWrapper>();
+            _awsCredentialsFactory = new Mock<IAwsCredentialsFactory>(); 
```

Comment:
Please be consistent with naming, the convention in other places is to suffix with `Mock`